FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -76,8 +76,8 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                         <dim unit="leaf">9</dim>
                                         <locus from="13r" to="21v"/>
                                             s.l. 1, no stub
-                                        <note>Contrary to <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">93</citedRange></bibl>, who considered <locus target="#13"/> separate from 
-                                            <ref target="#q3"/>. The text in this quire seemingly shows no breaks.</note>                      
+                                        <note>Contrary to <bibl><ptr target="bm:Marrassini1986Laurenziana"/><citedRange unit="page">93</citedRange></bibl>, 
+                                            who considered <locus target="#13"/> separate from <ref target="#q3"/>. The text in this quire seemingly shows no breaks.</note>                      
```

Comment:
add type to ref
`<ref type="quire" target="#q3"/>`

Reply:
Done.